agent/05_agent_configure.sh

```diff
@@ -245,7 +245,104 @@ function generate_cluster_manifests() {
   ansible-playbook -vvv \
           -e install_path=${SCRIPTDIR}/${INSTALL_CONFIG_PATH} \
           "${SCRIPTDIR}/agent/create-manifests-playbook.yaml"
+}
+
+function enable_load_balancer() {
+  if [[ "${PLATFORM_TYPE}" == "none" && "${NUM_MASTERS}" == "3" ]]; then
+    # setup haproxy as the load balancer
+    if [[ "${IP_STACK}" == "v6" ]]; then
+      # The "wildcard" is different depending on IP stack.
+      # See http://docs.haproxy.org/1.6/configuration.html#4.2-bind
+      export HAPROXY_WILDCARD="[::]"
+    else
+      export HAPROXY_WILDCARD="*"
+    fi
+
+    sudo dnf install -y haproxy
```

Comment:
For transient services like this, my personal preference would be to use a container if possible, to limit the impacts on the host system